adapters/repos/db/vector/hnsw/restore_test.go

```diff
@@ -50,7 +50,7 @@ func Test_RestartFromZeroSegments(t *testing.T) {
 	ef := 32
 	maxNeighbors := 32
 	dimensions := 20
-	vectors_size := 10000
+	vectors_size := 1000
 	queries_size := 1
 	vectors, _ := testinghelpers.RandomVecs(vectors_size, queries_size, dimensions)
```

Comment:
question: if we want to trim the time down, would it make sense to also cache these random vectors? I imagine there's no harm in reusing the same input across all tests for a single run.

Reply:
Most of the time isn't generating the random vectors but comes from the subsequent HNSW + brute force operations. These tests are more about correctness vs performance and don't need 10000 vectors.